packages/gatsby-recipes/src/cli/index.js

```diff
@@ -435,7 +435,11 @@ export default async ({
             </Text>
             <RecipesList
               setRecipe={async recipeItem => {
-                setRecipe(recipeItem.value.slice(0, -4))
+                if (recipeItem.value.endsWith(`.mdx`)) {
+                  setRecipe(recipeItem.value.slice(0, -4))
+                } else {
+                  setRecipe(recipeItem.value)
+                }
```

Comment:
```suggestion
          setRecipe={async ({value}) => {
               if (value.endsWith(`.mdx`)) {
                  setRecipe(value.slice(0, -4))
                } else {
                  setRecipe(value)
                }
```

Reply:
Yeah I dunno. Maybe `setRecipe` should do this instead? Or is this not that generic?